pxr/base/tf/testenv/fileUtils.cpp

```diff
@@ -234,6 +234,28 @@ TestTfIsFile()
     return true;
 }
 
+#if defined(ARCH_OS_WINDOWS)
+static bool
+TestTfJunction()
+{
+    cout << "Testing Windows junctions" << endl;
+
+    TF_AXIOM(TfMakeDir("junction_target"));
```

Comment:
This should be junction-target

Reply:
Good catch, thx

Reply:
Per https://dev.azure.com/PixarAnimationStudios/USD/_build/results?buildId=67&view=logs&j=a5e52b91-c83f-5429-4a68-c246fc63a4f7&t=61b979ba-27c9-588c-740e-a675d85b471e, does the CI run the tests on Windows? I've been running the test exe manually as the recommended ctest command line is reports no tests found.